torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3875,6 +3881,10 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                SkipInfo('TestOperatorSignatures', 'test_get_torch_func_signature_exhaustive'),
            ),
            sample_inputs_func=sample_inputs_unfold),
+    OpInfo('msort',
+           dtypes=all_types_and(torch.float16),
+           check_batched_gradgrad=False,
+           sample_inputs_func=sample_inputs_msort),
```

Comment:
Add a skip for test_out and add a comment above the skip with the error message.

Looks like msort does support bool, but only on a certain inputs? If that's the case then remove it from supported dtypes (like the current OpInfo) and skip the supported dtypes test for bool.

Are you able to run the msort opinfo tests locally?

Reply:
Yes I can actually run tests locally. and I think they can pass all of the tests now.